courses/models.py

```diff
@@ -68,6 +68,7 @@ def __str__(self):
 
     class Meta:
         unique_together = ('program', 'position_in_program',)
+        ordering = ('position_in_program',)
```

Comment:
I pulled this out into a new pull request: #1692. Maybe we can get that one merged first, and then rebase this PR on top of it.